library/ws-widget/collection/src/lib/_common/profile-card-stats/profile-card-stats.component.ts

```diff
@@ -44,11 +44,12 @@ export class ProfileCardStatsComponent implements OnInit {
     this.userInfo =  this.configSvc && this.configSvc.userProfile
     this.currentUserId = this.configSvc.unMappedUser.id
     if (this.userInfo) {
-      if (this.userInfo.firstName.length > 2) {
-        this.userName = this.userInfo.firstName
-      }
-      if (this.userInfo.firstName.length <= 2) {
-        this.userName = `${this.userInfo.firstName} ${this.userInfo.lastName}`
+      let userFstName = this.userInfo.firstName
+      const userNameFW = userFstName.split(' ')
```

Comment:
This code should be revreted back to previous one only, we have to apply logic for Timely nudges

Reply:
@venkykandagaddala , PR should be reviewed with required feature, please take care.